frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -511,46 +525,181 @@ function ReaderFooter:addToMainMenu(menu_items)
-                    self.settings.align = "center"
-                    self:updateFooterContainer()
-                    self:resetLayout(true)
-                    self:updateFooter()
-                    UIManager:setDirty(nil, "ui")
-                end,
+                text = _("Alignment"),
+                sub_item_table = {
+                    {
+                        text = _("Center"),
+                        checked_func = function()
+                            return self.settings.align == "center" or self.settings.align == nil
+                        end,
+                        callback = function()
+                            self.settings.align = "center"
+                            self:updateFooterContainer()
+                            self:resetLayout(true)
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                    {
+                        text = _("Left"),
+                        checked_func = function()
+                            return self.settings.align == "left"
+                        end,
+                        callback = function()
+                            self.settings.align = "left"
+                            self:updateFooterContainer()
+                            self:resetLayout(true)
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                    {
+                        text = _("Right"),
+                        checked_func = function()
+                            return self.settings.align == "right"
+                        end,
+                        callback = function()
+                            self.settings.align = "right"
+                            self:updateFooterContainer()
+                            self:resetLayout(true)
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                }
             },
             {
-                text = _("Left"),
-                checked_func = function()
-                    return self.settings.align == "left"
-                end,
-                callback = function()
-                    self.settings.align = "left"
-                    self:updateFooterContainer()
-                    self:resetLayout(true)
-                    self:updateFooter()
-                    UIManager:setDirty(nil, "ui")
-                end,
+                text = _("Text separator"),
+                sub_item_table = {
+                    {
+                        text = _("Vertical line") .. " (|)",
+                        checked_func = function()
+                            return self.settings.text_separator == "vertiacal_line" or self.settings.text_separator == nil
```

Comment:
verti**a**cal_line => `vertical_line` - or just `"bar"` ?
and `"none"` below instead of `"no_separator"`, `text_separator == "no_separator"` feels a bit redundant :)

Reply:
Technically called a `pipe` ;).

Reply:
Thanks.
I added another settings to switch between icons and letters.